### PR TITLE
README: update

### PR DESCRIPTION
**Pull request by nishinji**
- ~~Remove showcase (located on website,there are a lot of games that currently work)~~

### CODE REVIEW

**Review by CoffeeBrewer64 — COMMENTED** [review]
Why would you want to remove the showcase? Maybe cut it down by a few but it serves as a way to show off what the emulator can do.

Also, line 48, is that the correct one? I've been told before that it's now 2020/2021/2022 by Zangetsu.

**Review by CoffeeBrewer64 — CHANGES_REQUESTED** [review]
I think a link to the showcase which is on the website would be better so that the person can still see it if they want to.

File: README.md [hunk 1]
```diff
@@ -57,8 +46,8 @@ Vita3K is licensed under the **GPLv2** license. This is largely dictated by exte
 Please see [`building.md`](./building.md).
 
 ## Running
-Specify the path to a .vpk file as the first command line argument, or run `Vita3K --help` from the command-line for a full list of options.
-For more detailed instructions on running/installing games on all platforms, please read the **#info-faq** channel on our [Discord Server](https://discord.gg/MaWhJVH).
+Check our [quickstart](https://vita3k.org/quickstart) guide to make sure your computer meets the minimum requirements to run Vita3K.
```
Comment (CoffeeBrewer64) [inline README.md:49]: This is **really** petty but the link should cover the full "quickstart guide" instead of just "quickstart".

**Comment by Macdu** [comment]
I'm not entirely sure about the download counter badge given that it's reset after every update I believe.

**Comment by nishinji** [comment]
> I'm not entirely sure about the download counter badge given that it's reset after every update I believe.

What you are saying is correct and I recognize it, but I didn't pay much attention to it.
Deleted.

**Review by Macdu — DISMISSED** [review]

**Comment by scribam** [comment]
@Macdu can this be merged since it has been approved?

**Comment by nishinji** [comment]
VitaDB link is updated.
@scribam No one is objecting, so please merge :sob:.

**Review by scribam — APPROVED** [review]

**Pull request merged**